openwisp_radius/saml/urls.py

```diff
@@ -26,6 +26,11 @@ def get_saml_urls(saml_views=None):
             saml_views.AssertionConsumerServiceView.as_view(),
             name='saml2_acs',
         ),
+        path(
+            'additional-info/',
+            saml_views.LoginAdditionalInfoView.as_view(),
```

Comment:
I don't love the naming here, if we are asking only for the email here, why are we calling everything "additional info"?
I think we should call this view `LoginCollectEmailView`, the URL `collect-email/` and the template `collect-email.html`.

Reply:
I used a generic name in-case we used this view to ask for additional information. E.g., we could ask the user to verify the name received from the IdP.